pkg/filesharecontroller/selector/filter.go

```diff
@@ -0,0 +1,298 @@
+// Copyright (c) 2019 OpenSDS Authors. All Rights Reserved.
```

Comment:
Hope controller/selector/filter.go can be reused.

Reply:
selector i can't but filter we can